src/components/Button/Button.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types'
+import theme from '../../styles/theme'
+import {StyledButton} from "./Button.styles";
+
+const SIZES = ['small', 'medium', 'large']
+const COLORS = Object.keys(theme.colors);
+
+export const Button = ({...props}) => {
+    return <StyledButton {...props} />
+}
+
+Button.propTypes = {
+    color: PropTypes.oneOf(COLORS),
+    disabled: PropTypes.bool,
+    size: PropTypes.oneOf(SIZES)
```

Comment:
add `.isRequired` on mandatory props

Reply:
What are default value for if a field is required ?

Reply:
they are in defaultProps

Reply:
disabled must be required ?
@ttRmz